.github/workflows/review-code.yaml

```diff
@@ -0,0 +1,47 @@
+name: Review PR Diff
+
+on:
```

Comment:
Avoid adding unnecessary extra newlines in the git diff output. It may cause issues when parsing the diff.